executor/src/witgen/mod.rs

```diff
@@ -114,6 +114,12 @@ where
 
         rows.push(row_values.unwrap());
     }
+    // We re-compute the first row just in case we invalidly assumed unknown/unconstrained
+    // values are zero.
+    // TODO: do this properly.
```

Comment:
maybe open an issue so we don't forget?

Reply:
https://github.com/powdr-labs/powdr/issues/560